docs/apis-tools/migration-manuals/index.md

```diff
@@ -74,19 +74,19 @@ The [Camunda Java Client](/apis-tools/java-client/index.md) is now the official
 For more information on upgrading and migrating, see [migrate to Camunda Java Client](migrate-to-camunda-java-client.md).
 :::
 
-### Camunda Spring SDK
+### Spring Boot Camunda Starter
 
-The [Camunda Spring Boot SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) replaces the Spring Zeebe SDK. The SDK relies on the Camunda Java client, designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
+The [Spring Boot Camunda Starter](/apis-tools/spring-zeebe-sdk/getting-started.md) replaces the Spring Zeebe SDK. The SDK relies on the Camunda Java client, designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
```

Comment:
`Spring Zeebe SDK` - we had some discussions today with @megglos about this, as it seemed like we use the same artefact for the old and the new SDK. Is that correct?